zulip_bots/zulip_bots/lib.py

```diff
@@ -187,7 +187,17 @@ def run_message_handler_for_bot(lib_module, quiet, config_file, bot_name):
 
     state_handler = StateHandler()
 
+    # Set default bot_details, then override from class, if provided
+    bot_details = {
+        'name': bot_name.capitalize(),
+        'description': "",
+    }
+    bot_details.update(getattr(lib_module.handler_class, 'META', {}))
+
     if not quiet:
+        print("Running {} Bot:".format(bot_details['name']))
+        if bot_details['description'] != "":
+            print("\n\t{}".format(bot_details['description']))
```

Comment:
Nit: I find the partial indentation odd-looking:

```
Running Wikipedia Bot:

        Searches Wikipedia for a term and returns the top article.

            This plugin will allow users to directly search
            Wikipedia for a specific key term and get the top
            article that is returned from the search. Users
            should preface searches with "@mention-bot".
```
What do you think?

Reply:
You mean the name vs description vs usage indentation?

If so, I previously had the (short) description indented the same as the title, but chose this layout to illustrate the 3 components. I'm happy to adjust it back, though it's just shown on the console when running the bot.

Reply:
I think the components are already separated by the empty lines between them, so we should be fine using only one or two indentation levels.